.travis.yml

```diff
@@ -37,6 +37,11 @@ env:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
 install:
+  #  FIXME:
+  - git clone --branch 13.0-add_account_fiscal_position_allowed_journal_tbi
+    https://github.com/acsone/account-financial-tools.git PR937
+  - rm -rf account_fiscal_position_allowed_journal && cp -ar
+    PR937/account_fiscal_position_allowed_journal .
```

Comment:
https://github.com/OCA/account-financial-tools/pull/937 is merged, can you remove this?